master/reference/node/configuration.md

```diff
@@ -51,6 +51,22 @@ In addition to the above, `{{site.nodecontainer}}` also supports [the standard F
 >
 {: .alert .alert-info}
 
+### Node name determination
+
+The `{{site.nodecontainer}}` must know the name of the node on which it is running. The node name is used to
+retrieve the [Node resource](../calicoctl/resources/node) configured for this node if it exists, or to create a new node resource representing the node if it does not. It is
+also used to associate the node with per-node [BGP configuration](../calicoctl/resources/bgpconfig), [felix configuration](../calicoctl/resources/felixconfig), and endpoints.
+
+When launched, the `{{side.nodecontainer}}` container uses the following methods in order to determine
+what name to use:
+
+1. The value specified in the `NODENAME` environment variable, if set.
+1. The value specified at `/var/lib/calico/nodename` if it exists.
```

Comment:
Is this a file? It looks like it might be. In that case, I think "at" --> "in"

Not to be too detailed, but personally, I would wonder just what this file looks like. It might be good to include an example and perhaps some more details about the file. Is it a file they can or should manually modify?

Reply:
Yeah, this is a file on disk. 

I'll see what makes sense to document without making it too verbose... people shouldn't really be manually editing this file so might be best to leave it for now.